.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+14.17.3
```

Comment:
Since this is the `.nvmrc` file, isn't this more of the "preferred" node version to use for the project instead of the lowest supported version? As long as it's within the engine range it should be fine right?

Reply:
`lts/fermium` points to `14.19.3` and apparently docker doesnt have a cypress image for https://hub.docker.com/r/cypress/base/tags?page=1&name=14.19.3